SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -52,12 +52,20 @@
     PhysicsTablesDirectory = cms.string('PhysicsTables'),
     StorePhysicsTables = cms.bool(False),
     RestorePhysicsTables = cms.bool(False),
-    CheckOverlap = cms.untracked.bool(False),
+    CheckGeometry = cms.bool(False),
```

Comment:
@civanch why this and other verbosity flags are moved from untracked to tracked? What is the rationale behind it?

Reply:
@fabiocos, "verbosity" parameters were added by different peoples and historically some are tracked, another untracked. The way how these parameters are used is well defined: only for debuging. They mainly are inside this base python file. Likely some may be removed, another may stay "tracked".